examples/gno.land/p/demo/mux/query.gno

```diff
@@ -0,0 +1,74 @@
+package mux
+
+import "strings"
+
+type UrlQuery map[string][]string
```

Comment:
Maybe if familiarity with Go would be desired it could instead be called [Values](https://github.com/golang/go/blob/b8f83e22703ee23d49d95154449ce7066402d5c9/src/net/url/url.go#L916) or `QueryValues` for example